### PR TITLE
Works and About-me section

### PR DESCRIPTION
**Pull request by KareemWilson**
In this Branch I : 

1. started to build the "Projects" section and "Aboutme" section.
2. made some interactive buttons
3. considered the Mobile-first approach when dealing with grid layout so I made this project more responsive.
4. committed all changes one by one.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -12,35 +12,172 @@
...
+            <a
+              href="https://www.instagram.com/code.afternoon/"
+              class="fa fa-instagram"
+            ></a>
+          </div>
+        </section>
+      </div>
+      <section id="works" class="works-section">
+        <div class="section-container">
+          <h1 class="sectionTitle">My Recent Works</h1>
```
Comment (mugishasam123) [inline index.html]: - [ ] Kindly respect the order of headings to improve the accessibility of your pages .in this case you need to have only one  `h1` element in the whole document .you got this 👍

File: index.html [hunk 2]
```diff
@@ -12,35 +12,172 @@
...
+          </div>
         </div>
       </section>
+      <div class="aboutbg">
+        <img src="./utils/icons/aboutme1desktop.svg" alt="" class="aboutbg1">
+        <img src="./utils/icons/aboutmemobile1.svg" alt="" class="aboutbg1mobile">
+        <section id="About" class="aboutme-section">
+          <div class="section-container">
+            <div class="aboutHeader">
+              <h1 class="sectionTitle">About Me</h1>
```
Comment (mugishasam123) [inline index.html]: - [ ] Kindly you need to use only one `h1` element to improve you're page's accessibility.

File: index.html [hunk 3]
```diff
@@ -12,35 +12,172 @@
...
+              <p class="bioAboutme"> I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don't hestiate to contact me.</p>
+              <button class="resumeBtn">Get My Reusme</button>
+              <img src="./utils/icons/aboutme2desktop.svg" alt="" class="aboutbg2">
+              <img src="./utils/icons/aboutme2mobile.svg" alt="" class="aboutbg2mobile">
```
Comment (mugishasam123) [inline index.html]: - [ ] Kindly don't use `img` tag for the background instead use the image as a background of the section in the CSS select by setting the` background` property

**Review by mugishasam123 — CHANGES_REQUESTED** [review]
Hi  @KareemWilson  👋 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
alt
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Linters are passing 👍 
- Good commit messages ✔️ 
- Nice use of git-flow 🚀 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
[Optional]I would recommend you make the getting started section of the README file more descriptive and neat by for example showing setup instructions, etc to make your project more professional.
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by KareemWilson** [comment]
I changed the background code to be just in CSS and made a unique `<h1>` tag to improve my page's accessibility.  

Thank you in advance

**Review by Ismailco — APPROVED** [review]
### Hi @KareemWilson, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

cr-approve


## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**